test123.js

```diff
@@ -1 +1,3 @@
 console.log('Hello World');
+
+console.log('my unique string for realz');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=8530993)
```suggestion
console.log("my unique string for realz");
```